src/huggingface_hub/repository.py

```diff
@@ -852,6 +882,21 @@ def commit(self, commit_message: str, track_large_files: bool = True):
 
         """
 
+        files_to_stage = files_to_be_staged(".", folder=self.local_dir)
+
+        if len(files_to_stage):
```

Comment:
nit: I think we can directly do `if files_to_stage`

Reply:
Indeed!